src/engine/video/video.cpp

```diff
@@ -404,17 +351,11 @@ void VideoEngine::SetDrawFlags(int32_t first_flag, ...)
 
 void VideoEngine::Clear()
 {
-    //! \todo glClearColor is a state change operation. It should only be called when the clear color changes
```

Comment:
I moved the 'glClearColor' call to an initialization function.  The clear color was never changed at run-time anyways.

In addition, there was a subtle viewport bug where the cached viewport variables would become out of sync with the viewport itself.

So, the only 'glViewport' call should now be in the 'SetViewport' function.  I also removed the viewport update from the 'Clear' function.